packages/uxpin-merge-cli/src/program/command/init/getInitCommandSteps.ts

```diff
@@ -0,0 +1,75 @@
+import { copySync, existsSync, PathLike } from 'fs-extra';
+import { resolve } from 'path';
+import { printLine } from '../../../utils/console/printLine';
+import { PrintColor } from '../../../utils/console/PrintOptions';
+import { InitProgramArgs } from '../../args/ProgramArgs';
+import { getProjectRoot } from '../../args/providers/paths/getProjectRoot';
+import { Step } from '../Step';
+
+const RESOURCES_PATH:PathLike = '../../../resources';
+
+export interface DefaultFile {
+  source:PathLike;
+  target:PathLike;
+}
+
+const REQUIRED_DEPENDENCIES:string[] = [
+  'babel-loader',
+  '@babel/preset-env',
+  'prop-types',
+];
+
+const DEFAULT_CONFIG_FILES:DefaultFile[] = [
+  { source: 'uxpin.config.js', target: 'uxpin.config.js' },
+  { source: 'uxpin.webpack.config.js', target: 'uxpin.webpack.config.js' },
+  { source: 'UXPinWrapper.js', target: 'src/components/UXPinWrapper/UXPinWrapper.js' },
+];
+
+const EXAMPLE_COMPONENT:DefaultFile = {
+  source: 'Button',
+  target: 'src/components/Button',
+};
+
+export function getInitCommandSteps(args:InitProgramArgs):Step[] {
+  return [
+    { exec: copyDefaultFiles(args), shouldRun: true },
+    { exec: checkDependencies(), shouldRun: true },
+  ];
+}
+
+function copyDefaultFiles(args:InitProgramArgs):any {
+  const projectRoot:PathLike = getProjectRoot(args);
+  try {
+    // config files
+    DEFAULT_CONFIG_FILES.forEach((file) => {
+      const filePath:PathLike = `${projectRoot}/${file.target}`;
+      if (!existsSync(filePath)) {
+        copySync(resolve(__dirname, `${RESOURCES_PATH}/${file.source}`), filePath);
+        printLine(`✅ Successfully created ${filePath}`, { color: PrintColor.GREEN });
+      }
+    });
+
+    // default component
+    const componentPath:PathLike = `${projectRoot}/${EXAMPLE_COMPONENT.target}`;
+    if (!existsSync(componentPath)) {
+      copySync(resolve(__dirname, `${RESOURCES_PATH}/${EXAMPLE_COMPONENT.source}`), componentPath);
+      printLine(`✅ Successfully created example component in ${componentPath}`, { color: PrintColor.GREEN });
+    }
+  } catch (error) {
+    printLine('🛑 There was an error while copying default config files. Please try again.', { color: PrintColor.RED });
+    throw new Error(error.message);
+  }
+}
+
+function checkDependencies():any {
+  REQUIRED_DEPENDENCIES.forEach((library) => {
+    try {
+      require.resolve(library);
```

Comment:
I was testing this with Matthew and we did following steps

```
$ npx create-react-app test
$ cd test
$ yarn add -D @uxpin/merge-cli@2.7.2-dev.27
(I published package based on your branch to test)

$ npx uxpin-merge init

✅ Required babel-loader dependency found.
✅ Required @babel/preset-env dependency found.
✅ Required prop-types dependency found.

$ npx uxpin-merge --disable-tunneling

ERROR in multi ./node_modules/@uxpin/merge-cli/src/steps/serialization/component/presets/jsx/compile/globals/__uxpinParsePreset.js ./.uxpin-merge/presets-ed9f37ab-3a2a-4dc8
-9324-29fe74c1d1a2.js                                                                 
Module not found: Error: Can't resolve 'babel-loader' in '/home/matthew/Documents/UXPin/Development/test/cli-test-app-5/test'                                               
 @ multi ./node_modules/@uxpin/merge-cli/src/steps/serialization/component/presets/jsx/compile/globals/__uxpinParsePreset.js ./.uxpin-merge/presets-ed9f37ab-3a2a-4dc8-9324-
29fe74c1d1a2.js main[1]     
```

We got all greens for dependencies, but encountered this error. On my local, it actually worked fine without issue, but didn't work on Matthew's env and he had to install `babel-loader`.
`checkDependencies` might not be working in a way we want. (It looks fine though...)

I wonder if we can do something smart in webpack config file side🤔 (like.. `require.resolve('babel-loader')` ? )

Reply:
I think the current dependency checks you have are actually checking that dependencies of the _merge-cli_ rather than the React library....

I see two options what we could do:

1. Make sure that the required dependencies are part of merge-cli (just add them to the cli project) and that are set so that it forces them to be installed when merge-cli is added to a React library. 
2. Make the dependency checks test the actual React library, and just notify the user they will need to install them.

Option 1 would be best, but I'm not sure exactly how we do this. Peer dependencies seem a step too far.

Reply:
@mzah how are you getting on with this? Any progress/blockers?

Reply:
hyhm not sure how dirty it is but when i pointed root from where it should start looking:
`require.resolve(projectRoot + '/node_modules/' + library);`
i got something like this when i tried to follow scenario described by hiro:
<img width="683" alt="Zrzut ekranu 2021-02-16 o 17 17 56" src="https://user-images.githubusercontent.com/914841/108091109-a6ffc380-707b-11eb-81e6-0aeeec3316d1.png">

and after adding babel-loader i could run experimental mode. Will think about it more but wanted to share it

Reply:
@mzah 

Thanks for the update! I have just tested this and it seems to work ok.

This is definitely better than what we had before. However, if we _can_ get option 1 working, then that would be even better, and prevent the user from having to run yarn install to add the babel-loader:

>1. Make sure that the required dependencies are part of merge-cli (just add them to the cli project) and that they are set so that it forces them to be installed when merge-cli is added to a React library.)

Reply:
Tbh i do not have solution for that atm
1) babel-loader seems to be part of merge-cli already:
https://github.com/UXPin/uxpin-merge-tools/blob/master/packages/uxpin-merge-cli/package.json#L96
and i can also find it in node_modules but we still get error if we try use it
2) i did try similar approach to this one which hiro used in here:
https://github.com/UXPin/uxpin-merge-tools/pull/236/commits/f54acd47a3b50d8cd35ff57b3d6cbae11197fa54#diff-720650752204c267b9aa4425c6ed0bb6edc3774f44ec2ad4879f68507abd99eeR6
but it also did help

maybe i do not see/understand somethign but at this point this one:
https://github.com/UXPin/uxpin-merge-tools/pull/233#discussion_r576957601
is only one which i have/know/see

Reply:
@mzah - I'm not sure how you tried with my approach, but what I was thinking is something like https://github.com/UXPin/uxpin-merge-tools/compare/init-command...init-command-experiemnt 

And we can get rid of this dependencies check because those exit as uxpin-merge dependencies :)

Reply:
yeah, I overcomplicated it i think. Created mergeCLIWebpackLoader etc -,-
your solution does work properly, i did merge it and removed dependency checks